espnet2/train/preprocessor.py

```diff
@@ -599,6 +600,7 @@ def __init__(
                         space_symbol=space_symbol,
                         non_linguistic_symbols=non_linguistic_symbols,
                         g2p_type=g2p_type,
+                        encode_kwargs=tokenizer_encode_conf[i],
```

Comment:
For a normal ASR task that uses multiple tokenizer, like 3 different tokenizers (char, bpe2000, bpe5000), and the `tokenizer_encode_conf` is set to default, will the index `i` of `tokenizer_encode_conf` out of bound?

Reply:
Yeah, I previously only considered the case for `src_tokenizer` and `tgt_tokenizer` and set the default value to be None. But thanks for pointing this out. I add another value check now.